doc_requirements.txt

```diff
@@ -3,4 +3,5 @@
 Sphinx!=3.1.0, !=4.1.0
 pydata-sphinx-theme>=0.6.1
 sphinx-panels>=0.5.2
+numpydoc==1.1
```

Comment:
Any reason for the pinned version?

Reply:
Ok sure if there are blockers. Otherwise as you can see from the other doc dependencies there is no pin. Just versions to skip.

Reply:
> Otherwise as you can see from the other doc dependencies there is no pin.

:+1: my vote would be to put this PR in before #15443, then remove the numpydoc pins in #15443.